charts/datahub/subcharts/datahub-mce-consumer/templates/deployment.yaml

```diff
@@ -101,10 +101,64 @@ spec:
               value: "{{ . }}"
             {{- end }}
             {{- end }}
+            # TODO: Remove DATAHUB_GMS_HOST/PORT after v0.9.6
             - name: DATAHUB_GMS_HOST
               value: {{ printf "%s-%s" .Release.Name "datahub-gms" }}
             - name: DATAHUB_GMS_PORT
               value: "{{ .Values.global.datahub.gms.port }}"
+            - name: ENTITY_REGISTRY_CONFIG_PATH
+              value: /datahub/datahub-mce-consumer/resources/entity-registry.yml
+            - name: EBEAN_DATASOURCE_USERNAME
+              value: "{{ .Values.global.sql.datasource.username }}"
+            - name: EBEAN_DATASOURCE_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: "{{ .Values.global.sql.datasource.password.secretRef }}"
+                  key: "{{ .Values.global.sql.datasource.password.secretKey }}"
+            - name: EBEAN_DATASOURCE_HOST
+              value: "{{ .Values.global.sql.datasource.host }}"
+            - name: EBEAN_DATASOURCE_URL
+              value: "{{ .Values.global.sql.datasource.url }}"
+            - name: EBEAN_DATASOURCE_DRIVER
+              value: "{{ .Values.global.sql.datasource.driver }}"
+            - name: ELASTICSEARCH_HOST
+              value: "{{ .Values.global.elasticsearch.host }}"
+            - name: ELASTICSEARCH_PORT
+              value: "{{ .Values.global.elasticsearch.port }}"
+            - name: SKIP_ELASTICSEARCH_CHECK
+              value: "{{ .Values.global.elasticsearch.skipcheck }}"
+            {{- with .Values.global.elasticsearch.useSSL }}
+            - name: ELASTICSEARCH_USE_SSL
+              value: {{ . | quote }}
+            {{- end }}
+            {{- with .Values.global.elasticsearch.auth }}
+            - name: ELASTICSEARCH_USERNAME
+              value: {{ .username }}
+            - name: ELASTICSEARCH_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: "{{ .password.secretRef }}"
+                  key: "{{ .password.secretKey }}"
+            {{- end }}
+            {{- with .Values.global.elasticsearch.indexPrefix }}
```

Comment:
why having with here is better than just using this as value 2lines below?

Reply:
This is consistent with gms is all.